scripts/st2bootstrap-deb.sh

```diff
@@ -653,8 +653,26 @@ EOT"
 }
 
 install_st2chatops() {
-  # Add NodeJS 6 repo
-  curl -sL https://deb.nodesource.com/setup_6.x | sudo -E bash -
+  INSTALLABLE_ST2CHATOPS_VERSION=$(apt show st2chatops | grep 'Version:' | sed 's/^Version: \([[:digit:]]\+\)\.\([[:digit:]]\).*/\1.\2')
+
+  # Check if the st2chatops version is >= 3.0
+  python -c <<EOF
+import sys
+if '${INSTALLABLE_ST2CHATOPS_VERSION}' >= '3.0':
+    sys.exit(0)
+else:
+    sys.exit(1)
```

Comment:
Python interpreter exits with 1 by default if there is a syntax error in a file or similar so we should use some other, more unique exit code which won't have false positives.